### DISCUSSION TITLE
Backend Not responding!!

### DISCUSSION THREAD

**Discussion by amalraj91** [amalraj91/footy-client#4] [Q&A] [answered]
I've hosted my backend in vercel through github and frontend in netlify, but the backend is not responding after deployment but it works well in localhost. Kindly help

**Comment by shibincreji** [comment]
Did you check your env variables while deploying your project?

**Reply by amalraj91** [reply]
Yes I did

**Reply by shibincreji** [reply, marked as answer]
I guess you've uploaded .env in .ignore file, check that out

**Reply by amalraj91** [reply]
Oops my bad it was in .ignore file. Thanks a lot
